articles/deploying-crowdstrike-with-fleet.md

```diff
@@ -65,7 +65,7 @@ Below is an explanation of what each of the macOS CrowdStrike Falcon payloads do
 - `crowdstrike-web-filter.mobileconfig` - Enable web filtering to monitor network traffic at the socket level.
 - `crowdstrike-full-disk-access.mobileconfig` - Grant full disk access to all CrowdStrike application processes using the CrowdStrike Apple Developer team identifier.
 
-[Download the CrowdStrike Falcon macOS Configuration Profiles](https://github.com/fleetdm/fleet/tree/main/assets/configuration-profiles)
+[Download the CrowdStrike Falcon macOS Configuration Profiles](https://github.com/fleetdm/fleet/tree/main/docs/solutions)
```

Comment:
@harrisonravazzolo looks like these links will need to be updated to match the new file structure